core-build/gulp-core-build/src/tasks/GulpTask.ts

```diff
@@ -229,11 +231,13 @@ export abstract class GulpTask<TASK_CONFIG> implements IExecutable {
           throw new Error('The task subclass is missing the "executeTask" method.');
         }
 
-        stream = this.executeTask(this.buildConfig.gulp, (result?: Object) => {
-          if (!result) {
+        stream = this.executeTask(this.buildConfig.gulp, (error?: string) => {
+          if (!error) {
             resolve();
+          } else if (typeof error === 'string') {
+            reject(new Error(error));
```

Comment:
I'll remove the non-string case.

---
In reply to: [124900299](https://github.com/Microsoft/web-build-tools/pull/256#discussion_r124900299) [](ancestors = 124900299)